File: 프로그래머스/unrated/181834. l로 만들기/l로 만들기.js
function solution(myString) {
    for(var i=0; i<myString.length; i++){
        if(myString[i] < 'l'){
            myString = myString.replace(myString[i],'l');
        }
    }
    
    return myString;
}


//replace 사용
function solution(myString) {
    return myString.replace(/[a-k]/g,'l');
}

File: 프로그래머스/unrated/181834. l로 만들기/l로 만들기.java
class Solution {
    public String solution(String myString) {
        String answer = "";
        for(int i=0; i < myString.length(); i++){
            if(myString.charAt(i) < 'l') answer += 'l';
            else answer += myString.charAt(i);
        }
        return answer;
    }
}

//replace 사용
//class Solution {
//    public String solution(String myString) {
        
//        return myString.replaceAll("[^l-z]","l");
        
//    }
//}